packages/client/package.json

```diff
@@ -6,7 +6,9 @@
   "author": "Jesse Hallett <jesse@sitr.us>",
   "license": "Apache-2.0",
   "scripts": {
+    "prestart": "cd ../common && yarn build",
```

Comment:
The new common package needs to be compiled so that the other packages can use it. I put in some script hooks here to automatically compile common when running `yarn start` from client.